lib/tasks/import.rake

```diff
@@ -25,11 +25,30 @@ namespace :import do
       whitehall_document_id: args.document_id,
       whitehall_migration: WhitehallMigration.create!,
       state: "pending",
-     )
+    )
 
     WhitehallDocumentImportJob.perform_later(whitehall_import)
     puts "Added whitehall document with ID:#{args.document_id} to the import queue"
 
     puts whitehall_migration_url(whitehall_import.whitehall_migration_id, host: Plek.new.external_url_for("content-publisher"))
   end
+
+  desc "Import multiple documents from Whitehall Publisher using Whitehall's internal document ID e.g. import:whitehall_documents[\"123 456 789\"]"
```

Comment:
V minor but should `ID` be plural as we're referencing importing multiple docs and so multiple ID's?

Reply:
Good spot Peter - thanks!